### DISCUSSION TITLE
Lots of failed connections

### DISCUSSION THREAD

**Discussion by palfreveea** [community/community#66411] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Are other people seeing lots of failed connections today? My typical build makes dozen of github calls and and I'm seeing all sorts of random failed connections.  Their status page claims everything is fine thought, which cannot be right.

E.g. fatal: unable to access 'https://github.com/libtom/libtommath.git/': Failed to connect to github.com port 443: Connection refused

No sign of a "raise a support case" option but maybe we didn't pay for that.

**Comment by joske** [comment]
I had lots of github issues today with git operations timing out. Github status page says everything is A-OK which it isn't. I have the same issues on different machines/OSes.

**Reply by joske** [reply]
Next day, and still same issues

**Comment by konradasb** [comment]
Same issues here (related https://github.com/orgs/community/discussions/66444)
